apps/app_rpt/rpt_config.c

```diff
@@ -855,21 +855,21 @@ void load_rpt_vars(int n, int init)
 
 	RPT_CONFIG_VAR_INT(hangtime, "hangtime");
 	if (!val) {
-		rpt_vars[n].p.hangtime = (ISRANGER(rpt_vars[n].name) ? 1 : HANGTIME);
+		rpt_vars[n].p.hangtime = HANGTIME;
 	}
 	if (rpt_vars[n].p.hangtime < 1) {
 		rpt_vars[n].p.hangtime = 1;
 	}
 
 	RPT_CONFIG_VAR_INT(althangtime, "althangtime");
 	if (!val) {
-		rpt_vars[n].p.althangtime = (ISRANGER(rpt_vars[n].name) ? 1 : HANGTIME);
+		rpt_vars[n].p.althangtime = HANGTIME;
 	}
 	if (rpt_vars[n].p.althangtime < 1) {
 		rpt_vars[n].p.althangtime = 1;
 	}
 
-	RPT_CONFIG_VAR_INT_DEFAULT(totime, "totime", (ISRANGER(rpt_vars[n].name) ? 9999999 : TOTIME));
+	RPT_CONFIG_VAR_INT_DEFAULT(totime, "totime", TOTIME);
```

Comment:
This one was a mistake on my part.  We were supposed to be limiting the range of the value not checking for RANGER.  The current main branch has the correct code here.